namespaces/referencing-namespace.md

```diff
@@ -0,0 +1,194 @@
+# Referencing Namespace
+
+This namespace defines elements and attributes that may be used to provide the ability to reference and classify elements and namespaces.
+
+This document includes the elements as defined in the Refract specification.
+
+## Referencing and Identifying Elements
+
+These elements provide the means to identify elements with unique or reused qualifiers.
```

Comment:
So, something needs to be stated that these are actually reserved attributes for namespacing, right. Your examples show them in attributes, but you do not explicitly declare it.